config/develop/bridgeserver2.yaml

```diff
@@ -1,7 +1,9 @@
 template_path: bridgeserver2.yaml
 stack_name: bridgeserver2-develop
 depedencies:
-  - develop/bridgeserver2-db
+  - common/bridgeserver2-common.yaml
+  - common/bridgeserver2-common-db.yaml
```

Comment:
i don't think you need this line because I believe sceptre handles implicit stack dependencies.  you already have a dependency on this from  `develop/bridgeserver2-db.yaml`.  Same with files under prod and uat.